library/silverpeas

```diff
@@ -1,6 +1,9 @@
+# This file is generated via https://github.com/Silverpeas/docker-silverpeas-prod/blob/master/generate-docker-library.sh
 Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
 GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
 
-Tags: 6.0, latest
-GitCommit: 80bd9752fac5793594ddc80943ecdcf88e25f3b1
+Tags: 6.0.1
```

Comment:
Did you want to keep the `latest` for `6.0.1`?  It is the default image if users fail to specify a tag: ie. `docker run silverpeas`.

Reply:
Friendly ping @mmoqui

Reply:
Ping @mmoqui :cold_sweat:

Reply:
Oops, sorry. I was in vacation for 3 weeks.
Indeed, I want to link the latest to 6.0.1.

Reply:
I just updated the file so that 6.0.1 is tagged as the latest